docs/RELEASE-PROCESS.md

```diff
@@ -0,0 +1,87 @@
+# Release process
+
+## Determine the new version
+Extension versions may not have suffixes like "-rc1".
+We are incrementing patch versions to differentiate release candidate builds.
+
+For example, the 1.0 release will start with a build at version 1.0.0.
+Version 1.0.0 is, at first, a release candidate.
+If changes are needed, we move to the next build at version 1.0.1.
+Version 1.0.1 is also a release candidate at first.
+If no changes are needed, 1.0.1 becomes the first public release of 1.0.
+
+## Set the new version throughout
+You must update:
+
+- config.js
+- package.json
+- public/manifest.json
+
+## Add a changelog section
+Here's the [changelog](changelog.md).
+If there is an "Unreleased changes" section, rename it to the new version.
+Do not include a "v" prefix in the heading.
+
+Add any other user-facing changes since the last release that aren't entered.
+Suggested sections:
+
+- Breaking changes
+- New features
+- Bug fixes
+- Little things
+- Documentation changes
+
+Items may include a link to GitHub issues if there is a relevant one, e.g. in bug fixes.
+We especially encourage linking externally reported issues.
+
+If you feel like it, move one changelog item to a section on top titled "Spotlight change."
+
+## Propose the new version
+Open a pull request (PR) for the version changes and changelog update, ideally separate from
+any other changes.
+
+When this PR gets merged into `master`, the chosen version permanently associated with the merge
+commit.
+
+## Prepare an unsigned archive
+Clean the output directory, build, and package:
+
+```sh
+VERSION=<VERSION>
+rm -rf dist/
+yarn install --frozen-lockfile
+yarn buildProd
+zip -r oasis-wallet-$VERSION-$(git rev-parse --short HEAD).zip dist/
+```
+
+## Add a pre-release on GitHub
+Create a tag and create a
+[new release](https://github.com/oasisprotocol/oasis-wallet-ext/releases/new) on GitHub.
+
+TODO: standards on GitHub release title
+
+Our changelog is a file under version control, so all we do is put the link in the description:
+
+```md
+_Note: This is an internal release candidate._
+
+See the [changelog](docs/changelog.md) for what's new in this release.
+```
+
+Add the unsigned archive as an asset.
+
+Check the box to indicate that this is a pre-release.
+
+## Submit crx for review
+
+(Contact our web store account admins for this step.)
```

Comment:
Since these instructions are already primarily intended for wallet maintainers, I guess we can just write the exact thing so we have everything in one place.
Something like:
Go to [Chrome Web Store Developer Dashboard](https://chrome.google.com/webstore/devconsole/), browse to the _Oasis Wallet_ item, select _Package_ in the side bar and choose _Upload new package_.

Reply:
gonna paste that in, thanks